### DISCUSSION TITLE
RaspberryMatic-Fehler und Servicemeldungen per Home Assistant auf das iPhone pushen

### DISCUSSION THREAD

**Discussion by freigeistmaschine** [SukramJ/aiohomematic#1902] [Q&A]
Ich möchte gerne alle Informationen wie „Fehler“, „Gerät nicht erreichbar“ und „Batterie leer“ von meiner RaspberryMatic – also sowohl Alarme als auch Servicemeldungen – über Home Assistant auf mein iPhone gepusht bekommen.

Dafür habe ich diese zwei Blueprints importiert, aber die Fehler werden leider nicht auf mein Handy gesendet:

Blueprints:
**Support for persistent notifications for unavailable devices:** 
**Show device errors:**

**Comment by SukramJ** [comment]
Warum sollte auch eine Nachricht auf dein Handy gesendet werden? Der BP erzeugt eine Meldung, die in HA angezeigt wird.

**Reply by freigeistmaschine** [reply]
kann mir jemand helfen, was ich machen muss um diese 1zu1 auf das iPhone weiterzuleiten?

**Reply by SukramJ** [reply]
In diesem Repository geht es um die Integration in HA.
Für allgemeine HA Themen solltest Du einerseits die [HA Dokumentation](https://www.home-assistant.io/docs) lesen, und oder einschlägige Foren besuchen:
(DE): https://community.simon42.com/c/home-assistant-automationen
(EN): https://community.home-assistant.io/
